lib/support/startVideo.js

```diff
@@ -7,7 +7,7 @@ const webdriver = require('selenium-webdriver'),
 
 function getFFMpegArgs(mpegPath, options) {
   // calculate the viewport, lets spend some time and make this better
-  let size = '1366x708';
+  let size = '1280x1024';
```

Comment:
This should be 1280x964, right? Shaving off 60px for browser ui (window chrome). Could be cleaner to do
```
const size = options.viewPort || '1280x1024';
```
and drop the if-statement to shave off the 60px just in one place.